File: dmn-core/src/test/resources/tck/cl3/expected/java/dmn/0004-lending/BureauCallTypeTableRuleOutput.java

import java.util.*;

@javax.annotation.Generated(value = {"decisionTableRuleOutput.ftl", "BureauCallTypeTable"})
public class BureauCallTypeTableRuleOutput extends com.gs.dmn.runtime.RuleOutput {
    private String bureauCallTypeTable;

    public BureauCallTypeTableRuleOutput(boolean matched) {
        super(matched);
    }

    public String getBureauCallTypeTable() {
        return this.bureauCallTypeTable;
    }
    public void setBureauCallTypeTable(String bureauCallTypeTable) {
        this.bureauCallTypeTable = bureauCallTypeTable;
    }

    @Override
    public boolean equals(Object o) {
        if (this == o) return true;
        if (o == null || getClass() != o.getClass()) return false;

        BureauCallTypeTableRuleOutput other = (BureauCallTypeTableRuleOutput) o;
        if (this.getBureauCallTypeTable() != null ? !this.getBureauCallTypeTable().equals(other.getBureauCallTypeTable()) : other.getBureauCallTypeTable() != null) return false;

        return true;
    }

    @Override
    public int hashCode() {
        int result = 0;
        result = 31 * result + (this.getBureauCallTypeTable() != null ? this.getBureauCallTypeTable().hashCode() : 0);

        return result;
    }

    public String toString() {
        StringBuilder result_ = new StringBuilder("(matched=" + isMatched());
        result_.append(String.format(", bureauCallTypeTable='%s'", bureauCallTypeTable));
        result_.append(")");
        return result_.toString();
    }
}

File: dmn-core/src/test/resources/tck/cl3/expected/kotlin/dmn/0004-lending/BureauCallTypeTableRuleOutput.kt

@javax.annotation.Generated(value = ["decisionTableRuleOutput.ftl", "BureauCallTypeTable"])
class BureauCallTypeTableRuleOutput(matched: Boolean) : com.gs.dmn.runtime.RuleOutput(matched) {
    @com.fasterxml.jackson.annotation.JsonProperty("BureauCallTypeTable")
    var bureauCallTypeTable: String? = null

    override fun equals(o: Any?): Boolean {
        if (this === o) return true
        if (javaClass != o?.javaClass) return false

        val other = o as BureauCallTypeTableRuleOutput
        if (if (this.bureauCallTypeTable != null) this.bureauCallTypeTable != other.bureauCallTypeTable else other.bureauCallTypeTable != null) return false

        return true
    }

    override fun hashCode(): Int {
        var result = 0
        result = 31 * result + (if (this.bureauCallTypeTable != null) this.bureauCallTypeTable.hashCode() else 0)

        return result
    }

    override fun toString(): String {
        val result_ = StringBuilder("(matched=" + isMatched())
        result_.append(String.format(", bureauCallTypeTable='%s'", bureauCallTypeTable))
        result_.append(")")
        return result_.toString()
    }
}
